sos/presets/redhat/__init__.py

```diff
@@ -36,9 +36,13 @@
 RHOCP_DESC = "OpenShift Container Platform by Red Hat"
 RHOCP_OPTS = SoSOptions(
     skip_plugins=['cgroups'], container_runtime='crio', no_report=True,
-    log_size=100,
+    log_size=100, enable_plugins=['openshift','openshift_ovn'],
```

Comment:
Nitpick: ad space after comma after 'openshift' (`flake8 complains).

Reply:
Apologies for missing it. I'll push a fix today